

错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=5, tm_sec=20, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=5, tm_sec=24, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=8, tm_sec=5, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=8, tm_sec=5, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=45, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=46, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=46, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=51, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=51, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=51, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=51, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=52, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=53, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=53, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=53, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=53, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=53, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=34, tm_sec=53, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=7, tm_mday=16, tm_hour=22, tm_min=41, tm_sec=29, tm_wday=6, tm_yday=197, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=8, tm_mday=1, tm_hour=16, tm_min=35, tm_sec=43, tm_wday=1, tm_yday=213, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=8, tm_mday=1, tm_hour=16, tm_min=35, tm_sec=50, tm_wday=1, tm_yday=213, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
错误发生时间:time.struct_time(tm_year=2023, tm_mon=8, tm_mday=1, tm_hour=16, tm_min=35, tm_sec=52, tm_wday=1, tm_yday=213, tm_isdst=0)	错误文件地址:main.py	错误函数:function main()	当前程序的第＊次错误:1
